x/crosschain/keeper/cctx_gateway_observers.go

```diff
@@ -75,7 +75,7 @@ func (c CCTXGatewayObservers) InitiateOutbound(
 	}()
 	if err != nil {
 		// do not commit anything here as the CCTX should be aborted
-		config.CCTX.SetAbort(err.Error())
+		config.CCTX.SetAbort("Aborted due to an internal error", err.Error())
```

Comment:
```suggestion
		config.CCTX.SetAbort("aborted due to an internal error", err.Error())
```

I think we can keep Go error standard that is to keep all lowercase